### DISCUSSION TITLE
App Crashed

### DISCUSSION THREAD

**Discussion by AlephBaa** [vfsfitvnm/frida-il2cpp-bridge#278] [Q&A]
Tracing some Class functions results in crashing the app.
for example:
```
Il2Cpp.perform(() => {

    const assem = Il2Cpp.Domain.assembly('Assembly-CSharp');
    const ExmplClass = assem.image.class('ExampleClass');
	
    Il2Cpp.trace()
	.classes(ExmplClass)
	.and()
        .attach("detailed");
});
```

after tracing some functions, results in:

FATAL EXCEPTION: UnityMain
Process: com.example.example, PID: 26874
java.lang.Error: *** *** *** *** *** *** *** *** *** *** *** *** *** *** *** ***
Version '2021.3.11f1 (0a5ca18544bf)', Build type 'Release', Scripting Backend 'il2cpp', CPU 'armeabi-v7a'
Build fingerprint: 'xiaomi/cactus/cactus:9/PPR1.180610.011/V11.0.8.0.PCBMIXM:user/release-keys'
Revision: '0'
ABI: 'arm'
Timestamp: 2023-04-02 13:13:19+0430
pid: 26874, tid: 26929, name: UnityMain  >>> com.example.example <<<
uid: 10164
signal 11 (SIGSEGV), code 1 (SEGV_MAPERR), fault addr 0x4
Cause: null pointer dereference
    r0  00000004  r1  a6beeefb  r2  00001204  r3  00000000
    r4  81dfc6c8  r5  00000001  r6  00000001  r7  81dfc740
    r8  a3fdafd8  r9  81dfc770  r10 81dfc6c8  r11 81dfc6d8
    ip  00000000  sp  81dfc6d8  lr  8acf0304  pc  7ed688b8

backtrace:
      #00 pc 004178b8  /data/app/com.example.example-H-cHzgW8kgNBnmjJBIYEdQ==/lib/arm/libil2cpp.so (jinfo_get_method) (BuildId: 1258ce5b89a8aab5f270238d7504cf4597ead74a)
      #01 pc 004ed300  /data/local/tmp/re.frida.server/frida-agent-32.so

        at libil2cpp.jinfo_get_method(jinfo_get_method:0)
        at frida-agent-32.0x4ed300(Native Method)

**Comment by vfsfitvnm** [comment]
Uhm, it looks like a Frida issue. What's the app id?
